packages/ui5-application-writer/test/options.test.ts

```diff
@@ -129,4 +129,28 @@ describe('UI5 templates', () => {
             }
         });
     });
+
```

Comment:
Instead of adding a new test, could we add a namespace to the app id of the `baseAppConfig` so that we are checking this case for all options?

Reply:
Happy to make this change. The new test was introduced as there were alot of changes to the snapshots if the baseAppConfig ID was changed, wanted to keep this PR to a minimum amount of changes.